podcasts/Referrals/ReferralSendPassView.swift

```diff
@@ -66,6 +80,6 @@ struct ReferralSendPassView: View {
 
 #Preview("With Passes") {
     Group {
-        ReferralSendPassView(viewModel: ReferralSendPassModel(offerInfo: ReferralsOfferInfoMock()))
+        ReferralSendPassView(viewModel: ReferralSendPassModel(offerInfo: ReferralsOfferInfoMock(), referralURL: URL(string: "https://pocketcasts.com")))
```

Comment:
Same here about the URL

Reply:
Good catch, but the URL here will be something defined by the server for each user... so this was a temporary use. But I updated it [here](https://github.com/Automattic/pocket-casts-ios/pull/2166/commits/0f55838c8c48374c404731227546613ec96d25bf) to use the constant for now.